src/lib.rs

```diff
@@ -59,7 +57,3 @@ pub mod vm;
 pub use arch::*;
 use lazy_static::lazy_static;
```

Comment:
Can we also remove lazy_static?

Reply:
This is already removed in the second commit in this pull request :wink: The crate itself cannot be removed as it is used in the library.

Reply:
Ah, I oversaw it